roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1599,11 +1604,13 @@ def __init__(self, role, filename, local_facts,
 
         try:
             # ansible-2.1
-            # pylint: disable=too-many-function-args
+            # pylint: disable=too-many-function-args,invalid-name
             self.system_facts = ansible_facts(module, ['hardware', 'network', 'virtual', 'facter'])
-        except TypeError:
-            # ansible-1.9.x,ansible-2.0.x
-            self.system_facts = ansible_facts(module)
+            for (k, v) in self.system_facts.items():
+                self.system_facts["ansible_%s" % k.replace('-', '_')] = v
+        except UnboundLocalError:
+            # ansible-2.2
+            self.system_facts = get_all_facts(module)['ansible_facts']
```

Comment:
@abutcher Looking through them, I'm still pretty sure the quick installer doesn't really use any of the ones that changed